examples/ppo_sentiments.py

```diff
@@ -48,7 +48,7 @@ def reward_fn(samples: List[str]) -> List[float]:
     imdb = load_dataset("imdb", split="test")
     val_prompts = [" ".join(review.split()[:4]) for review in imdb["text"]]
 
-    return trlx.train(
+    trlx.train(
```

Comment:
No it should return.

Reply:
It breaks RayTune:
```
ray.exceptions.RayTaskError(LiveError): ray::ImplicitFunc.train() (pid=1701167, ip=26.0.134.192, repr=main)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/trainable.py", line 367, in train
    raise skipped from exception_cause(skipped)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/function_trainable.py", line 335, in entrypoint
    return self._trainable_func(
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/function_trainable.py", line 652, in _trainable_func
    output = fn()
  File "/admin/home-guac/tmp/ayu/trlx/examples/ppo_sentiments.py", line 51, in main
    trlx.train(
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trlx.py", line 107, in train
    trainer.learn()
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trainer/accelerate_base_trainer.py", line 385, in learn
    results = self.evaluate()
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trainer/accelerate_base_trainer.py", line 224, in evaluate
    for prompts in tqdm(self.eval_dataloader, desc="Generating samples"):
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/tqdm/rich.py", line 116, in __init__
    self._prog.__enter__()
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/progress.py", line 1169, in __enter__
    self.start()
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/progress.py", line 1160, in start
    self.live.start(refresh=True)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/live.py", line 113, in start
    self.console.set_live(self)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/console.py", line 840, in set_live
    raise errors.LiveError("Only one live display may be active at once")
rich.errors.LiveError: Only one live display may be active at once
```

and every other example we have does not return the trainer.

Reply:
Got it.